dm/tests/_utils/run_downstream_cluster

```diff
@@ -34,12 +34,18 @@ EOF
 		--data-dir "$WORK_DIR/pd" &
 	# wait until PD is online...
 	i=0
-	while ! curl "http://$PD_ADDR/pd/api/v1/version"; do
+	while true; do
+		response=$(curl -s -o /dev/null -w "%{http_code}" "http://$PD_ADDR/pd/api/v1/version")
```

Comment:
validate curl code instead of curl success or not. since curl get success but may get `[PD:server:ErrServerNotStarted]server not started`